src/Backoffice/Employee/Domain/Actions/ListEmployeeAction.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\Domain\Actions;
+
+use Illuminate\Database\Eloquent\Collection;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+use Spatie\QueryBuilder\QueryBuilder;
+
+class ListEmployeeAction
+{
+    /**
+     * @return Collection<int, Employee>
+     */
+    public function execute(): Collection
+    {
+        /**
+         * @var Collection<int, Employee>
+         */
```

Comment:
A little bit more compact, useful if you have more than a few lines in the File
```suggestion
        /** @var Collection<int, Employee> */
```